src/main/java/com/iexec/worker/tee/scone/LasServicesManager.java

```diff
@@ -10,21 +12,20 @@
 import org.springframework.stereotype.Service;
 
 import javax.annotation.PreDestroy;
-
 import java.util.HashMap;
 import java.util.Map;
 import java.util.UUID;
 
 @Service
 @Slf4j
-public class LasServicesManager {
+public class LasServicesManager implements Purgeable {
```

Comment:
Could we add some javadoc to explain difference between `chainTaskIdToLasService` and `lasImageUriToLasService` ?
Maybe `lasServiceFromChainTaskId` could be a more accurate name for the first map ?
I mean that this name seems to better describe the field purpose.

Reply:
JavaDoc improved: https://github.com/iExecBlockchainComputing/iexec-worker/pull/471/commits/f25c29bece5382367a6631a2585aee262b325a82
Regarding the name: I don't find any difference between actual name (`chainTaskIdToLasService`) and your proposal (`lasServiceFromChainTaskId`). As I understand them, they tell the same thing: a LAS service is associated to a chain task ID. In both versions, it is never said whether a LAS service is associated with exactly one or more than one tasks. So that's the same for me.

Reply:
Thanks